ui/app/templates/components/calendar-widget.hbs

```diff
@@ -62,10 +74,10 @@
             />
           </button>
           <p data-test-display-year>
-            {{this.displayYear}}
+            {{this.getDisplayYear}}
```

Comment:
Small nit but this makes it seems like you're calling a function. I think `displayYear` may have more accurately described the property.

Reply:
ah yes! I meant to change this back to displayYear - `getDisplayYear` was just supposed to be a temporarily variable while I refactored how they were computed

thank you for calling out!

Reply:
+1